llama_index/download/dataset.py

```diff
@@ -72,9 +88,19 @@ def get_dataset_info(
             raise ValueError("Loader class name not found in library")
 
         dataset_id = library[dataset_class]["id"]
-        source_files = _get_source_files_list(
-            str(remote_source_dir_path), f"/{dataset_id}/{source_files_path}"
+
+        # get data card
+        raw_card_content, _ = get_file_content(
+            str(remote_dir_path), f"/{dataset_id}/card.json"
         )
+        card = json.loads(raw_card_content)
+        dataset_class_name = card["className"]
+
+        source_files = []
+        if dataset_class_name == "LabelledRagDataset":
+            source_files = _get_source_files_list(
```

Comment:
we only get source files if its a `LabelledRagDataset` ?

Reply:
Yeah, since with evaluating a `Rag` we need to enable a user to build a Rag over the source documents (otherwise we don't have a Rag to evaluate).

In contrast, with evaluating an `Evaluator` there is nothing needed to build over the source documents. In other words, all that is needed is the <query, answer, ground_truth_answer> for an evaluator to evaluate. (Then we evaluate this evaluators response lol)